build-tools/packages/build-cli/src/commands/generate/buildVersion.ts

```diff
@@ -0,0 +1,130 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable camelcase */
+
+import child_process from "child_process";
+import * as fs from "fs";
+import { getSimpleVersion, getIsLatest } from "@fluidframework/build-tools";
+import { Flags } from "@oclif/core";
+import { BaseCommand } from "../../base";
+
+export default class GenerateBuildVersionCommand extends BaseCommand<
```

Comment:
Can we add a doc comment to this with a brief overview (or link to one that lives elsewhere)?